ha/core/system_health/const.py

```diff
@@ -70,3 +70,37 @@ class HEALTH_EVALUATOR_CLASSES:
 # Confstore key attributes
 class CONFSTORE_KEY_ATTRIBUTES(Enum):
     STORAGE_SET_ID = "storage_set_id"
+
+EVENTS = [
```

Comment:
Similar stuff being added through PR #496 . Can we use one that would serve for both purposes?

Reply:
We will refactor this later when all code will be merged to main.